python/pyspark/mllib/linalg.py

```diff
@@ -440,7 +440,7 @@ def __init__(self, size, *args):
         values (sorted by index).
 
         :param size: Size of the vector.
-        :param args: Non-zero entries, as a dictionary, list of tupes,
+        :param args: Non-zero entries, as a dictionary, list of tuples,
```

Comment:
In scala doc, there is not seems to be a reference to non-zero entries. Is it OK to leave it as it is?
```
 * @param indices index array, assume to be strictly increasing.
 * @param values value array, must have the same length as the index array.
```